### ISSUE TITLE
Jivo + Swift

### ISSUE THREAD

**Issue by Sunrizzin** [JivoSite/MobileSdk#7]
Добрый день!

Подключаю в приложении на Swift. При запуске на вью ничего не появляется.
1) Конфиги прописал
2) Импорт в бридж файле сделал
3) Делегат подключил

Вот моя реализация методов в контроллере чата:

```
    override func viewDidLoad() {
        super.viewDidLoad()
        
        jivo = JivoSdk(web, "ru")
        
        jivo!.delegate = self
        
        jivo!.prepare()
    }
    
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        jivo!.start()
    }
```
В логах никаких сообщений нет.
Подскажите пожалуйста, в чём может быть проблема. 
Заранее спасибо!

**Comment by dimmetrius** [comment]
Добрый день. 

Проверьте следующие моменты:
1) папка html копируется в корень проекта
![Image1](https://jivo-dev.s3.amazonaws.com/mobile/sdk/GitHub/img/im1.png)
2) обратите внимание на опции при добавлении ресурса
![Image2](https://jivo-dev.s3.amazonaws.com/mobile/sdk/GitHub/img/im2.png)

3) в debug проверьте правильность подключенных ресурсов и строк
![Image3](https://jivo-dev.s3.amazonaws.com/mobile/sdk/GitHub/img/im3.png)

4) к webview можно подключиться на отладку в safari и посмотреть загрузку и логи

**Comment by Fedorsturov** [comment]
Добрый день! Получилось встроить JivoSDK в swift3?

**Comment by Sunrizzin** [comment]
Добрый! Да и давно уже. Работает как часы)

**Comment by Fedorsturov** [comment]
можете поделиться наработками? я пошел через WKUIDelegate и застопорился

**Comment by Sunrizzin** [comment]
Да, конечно.


1) Во вложении файл моей реализации



2) Папка html должна быть импортирована в корень проекта


Вот в принципе и всё)

**Comment by Fedorsturov** [comment]
огромное спасибо

**Comment by AndriiDoroshko** [comment]
@Sunrizzin Привет) У вас приходят сообщения от сервера? Ставлю бряку на `onEvent` и никогда не вызывается( Может вы разобрались в чем может быть беда.

**Comment by xpams** [comment]
@AndriiDoroshko получилось разобраться? У меня такая же проблема..

**Comment by AndriiDoroshko** [comment]
@xpams да, в HTML файлах стоит строчка, что бы ивенты в консоль кидать, удали ее

**Comment by leocassioo** [comment]
Hello everyone!
I solved my problem in the following section:
![captura de tela 2018-06-27 as 12 05 34](https://user-images.githubusercontent.com/16023720/41982688-71aa6ea0-7a02-11e8-804f-022910706f87.png)

I noticed that the htmlFile variable was null because of the indexFile that did not display the expected behavior.

Best regards

**Comment by igorkaryi** [comment]
привет. можете поделится реализацией для Swift?

**Comment by AndriiDoroshko** [comment]
@igorkaryi Проект комерческий, код дать не могу. Если есть проблема, могу попробовать помочь. А если нужен пример, то попробуй поиск по github сделать, наверняка есть.

**Comment by igorkaryi** [comment]
loader на вебвью появился. но просто белая страница

**Comment by AndriiDoroshko** [comment]
@igorkaryi Можешь пошарить проект? Я вечером посмотрю, сейчас основные задачи подъехали. Там еще есть log message, они что то говорят?

**Comment by Datastore24Kirill** [comment]
Кто-нибудь, киньте проект на Swift

**Comment by AndriiDoroshko** [comment]
@Datastore24Kirill не могу, все коммерческое.
Что у тебя не получается? Чем тебе пример ObjC не годится?

**Comment by AhmAbdallah** [comment]
> Добрый день!
> 
> Подключаю в приложении на Swift. При запуске на вью ничего не появляется.
> 
> 1. Конфиги прописал
> 2. Импорт в бридж файле сделал
> 3. Делегат подключил
> 
> Вот моя реализация методов в контроллере чата:
> 
> ```
>     override func viewDidLoad() {
>         super.viewDidLoad()
>         
>         jivo = JivoSdk(web, "ru")
>         
>         jivo!.delegate = self
>         
>         jivo!.prepare()
>     }
>     
>     override func viewWillAppear(_ animated: Bool) {
>         super.viewWillAppear(animated)
>         jivo!.start()
>     }
> ```
> В логах никаких сообщений нет.
> Подскажите пожалуйста, в чём может быть проблема.
> Заранее спасибо!

I faced this problem, after searching in the Russian documentation, your solution saved my time.
